craco.config.cjs

```diff
@@ -100,10 +125,35 @@ module.exports = {
           return true
         })
 
-      // We're currently on Webpack 4.x which doesn't support the `exports` field in package.json.
-      // Instead, we need to manually map the import path to the correct exports path (eg dist or build folder).
-      // See https://github.com/webpack/webpack/issues/9509.
-      webpackConfig.resolve.alias['@uniswap/conedison'] = '@uniswap/conedison/dist'
+      webpackConfig.resolve.plugins = webpackConfig.resolve.plugins.map((plugin) => {
```

Comment:
q: what's the difference between the resolve.plugins and the plugins we mapped through on ln 90-91?

Reply:
These are resolver-specific plugins, whereas the other plugins have to do with the global pipeline.